server/utils/commentidvalidator.util.js

```diff
@@ -0,0 +1,18 @@
+import isUUID from 'validator/lib/isUUID';
+import Response from './response.util';
+
+/**
+ * @description Checks if the commentid is valid
+ * @param {Object} req
+ * @param {Object} res
+ * @param {function} next
+ * @returns {Object} response
+ */
+const commentIdValidator = (req, res, next) => {
+  const { commentId } = req.params;
+
+  if (!isUUID(commentId, 4)) return Response(res, 400, 'Please enter a valid id.');
```

Comment:
Good test cases @cupidcoder, I will appreciate if you can explain what the function of this line is here.

Reply:
Thanks, @samorahno. This line ensures that the `commentId` passed in the parameter is a valid `UUID` string, which is what the database expects.

Reply:
@cupidcoder thank you for the swift response, kindly throw more light on what the `4` does, that is (commentId, 4)

Reply:
@samorahno `uuid` basically has 3 different versions (3,4,5). So we need to specify the appropriate version to validate against.

Reply:
Thanks